SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/OutgoingMessages.h

```diff
@@ -58,65 +58,75 @@ struct FReserveEntityIdsRequest : FOutgoingMessage
 
 struct FCreateEntityRequest : FOutgoingMessage
 {
-	FCreateEntityRequest(TArray<FWorkerComponentData>&& InComponents, const Worker_EntityId* InEntityId)
+	FCreateEntityRequest(TArray<FWorkerComponentData>&& InComponents, const Worker_EntityId* InEntityId, const TOptional<worker::c::Trace_SpanId>& SpanId)
```

Comment:
I don't think anything in this file is actually used anymore (except maybe the metrics or queries). I think I just forgot to delete it when removing the legacy connection handler.

Reply:
I guess easier just to leave this in and delete it when we want to remove?

Reply:
I might delete it as a drivby to the other stuff. This code is all vestigial so it doesn't do anything bad here other than make the pr longer.